### PR TITLE
openshift/telemeter: Use base images from the repository

### PR DESCRIPTION
**Pull request by marioferh**
Use base images from the repository

### CODE REVIEW

File: ci-operator/config/openshift/telemeter/openshift-telemeter-master.yaml [hunk 1]
```diff
@@ -15,9 +11,6 @@ images:
     ocp_4.15_base:
       as:
       - registry.ci.openshift.org/ocp/4.15:base
```
Comment (jan--f) [inline ci-operator/config/openshift/telemeter/openshift-telemeter-master.yaml:13]: Remove this too.

Comment (marioferh) [reply ci-operator/config/openshift/telemeter/openshift-telemeter-master.yaml]: done

**Comment by slashpai** [comment]
Shouldn't we update https://github.com/openshift/release/blob/master/ci-operator/config/openshift/telemeter/openshift-telemeter-release-4.15.yaml for release-4.15 branch to work?
Also https://github.com/openshift/release/blob/master/ci-operator/config/openshift/telemeter/openshift-telemeter-release-4.16.yaml same changes as master file?

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Pull request merged**